src/syscalls.csv

```diff
@@ -0,0 +1,468 @@
+# name, index, x86_64, arm, aarch64, mips64n32, mips64, mips, parisc, ppc64, ppc, s390, s390x, x32, x86. riscv64
+accept, 3, 43, 285, 202, 42, 42, 168, 35, 330, 330, PNR, PNR, 0x40000000 + 43, PNR, 202
```

Comment:
Why isn't the 0x40000000 offset included in the per-ABI offset for x32?

Reply:
dropped and moved it as part of the arch definition